project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.7.3
+sbt.version=1.7.2
```

Comment:
Unfortunately sbt 1.7.3 ships with scala-xml 2.x which causes dependency conflicts. Staying on 1.7.2 fixed that.